### ISSUE TITLE
eth1 is being used as control interface always and "docker start" fails when ansible playbook is run

### ISSUE THREAD

**Issue by abhi** [contiv/ansible#59]
I am seeing this issue when i run net_installer_demo script. The cfg.yml contains the following

```
CONNECTION_INFO:
     172.29.205.249:
       **control: p1p2**
       data: eth2
```

**Ansible output:**

```
TASK: [docker | copy systemd units for docker tcp socket settings] ************
ok: [node1]

TASK: [docker | start docker tcp socket service] ******************************
failed: [node1] => {"changed": true, "cmd": "sudo systemctl stop docker && sudo systemctl start docker-tcp.socket && sudo systemctl start docker", "delta": "0:00:00.164433", "end": "2016-02-03 03:51:51.044325", "rc": 1, "start": "2016-02-03 03:51:50.879892", "warnings": []}
stderr: Warning: docker.service changed on disk. Run 'systemctl daemon-reload' to reload units.
Warning: docker.service changed on disk. Run 'systemctl daemon-reload' to reload units.
Job for docker.service failed. See "systemctl status docker.service" and "journalctl -xe" for details.

FATAL: all hosts have already failed -- aborting       
```

**journal-xe output**

```
-- Unit serf.service has begun starting up.
Feb 03 04:07:41 contiv146 serf.sh[13880]: setting up iptables for mdns
Feb 03 04:07:41 contiv146 serf.sh[13880]: starting serf
Feb 03 04:07:43 contiv146 serf.sh[13880]: eth1 is not assigned a valid addr: bin boot dev etc home initrd.img lib lib64 lost+found media mnt opt proc root run sbin srv sys tmp usr var vmlinuz
Feb 03 04:07:43 contiv146 systemd[1]: serf.service: main process exited, code=exited, status=1/FAILURE
Feb 03 04:07:43 contiv146 systemd[1]: Unit serf.service entered failed state.
Feb 03 04:07:43 contiv146 systemd[1]: serf.service failed.
Feb 03 04:07:49 contiv146 /usr/sbin/irqbalance[1321]: irq 56 affinity_hint subset empty
Feb 03 04:07:53 contiv146 systemd[1]: serf.service holdoff time over, scheduling restart.
Feb 03 04:07:53 contiv146 systemd[1]: Started Serf.
-- Subject: Unit serf.service has finished start-up
-- Defined-By: systemd
```

**Comment by abhi** [comment]
@mapuri i have a fix which i used to move forward with the issue. do you want to check it out before @vijayvikrant goes ahead working on this.

**Comment by mapuri** [comment]
@abhinandanpb go for it, I didn't know you had a fix. I will assign this to you.

**Comment by mapuri** [comment]
fixed by #70
